flashflood/util.py

```diff
@@ -66,3 +70,33 @@ def overlaps(self, *args, **kwargs):
 
     def contains(self, *args, **kwargs):
         return False
+
+def delete_keys(bucket, keys):
+    def _delete(keys_to_delete):
+        assert 1000 >= len(keys_to_delete)
```

Comment:
is there a case where this triggered? Otherwise it seems excessive.

Reply:
Pull out the 1000 into a variable which describes it.

Reply:
That's the maximum number of keys allowed in an S3 batch delete request.

Will fix.